app/models/work-item-interface.ts

```diff
@@ -50,4 +52,20 @@ export interface WorkItemRecord {
 
   // When the item was created
   createdAt: Date;
+}
+
+/**
+ * Get the s3 URL to the STAC outputs directory for a work item.
+ * Optionally pass in a target URL in which case the URL returned will be the target URL
+ * resolved relative to the STAC outputs directory.
+ * e.g. s3://artifacts/abc/123/outputs/ with a targetUrl of ./catalog0.json or catalog0.json would resolve to
+ * s3://artifacts/abc/123/outputs/catalog0.json
+ * @param item - the returned URL will provide the path to the outputs for this work item
+ * @param targetUrl - URL to resolve against the base outptuts directory 
+ * @param isAggregate - include the word aggregate in the URL
+ * @returns - the path to the STAC outputs directory (e.g. s3://artifacts/abc/123/outputs/) or the full path to the target URL
+ */
+export function getStacLocation(item: { id: number, jobID: string }, targetUrl = '', isAggregate = false): string {
+  const baseUrl = `s3://${env.artifactBucket}/${item.jobID}/${isAggregate ? 'aggregate-' : ''}${item.id}/outputs/`;
```

Comment:
The isAggregate check and special logic makes me think we might have this logic in another place. Is it possible to define the logic for the aggregate filename in one place in the code? Or is that logic with the file naming in the python service library?

Reply:
I didn't see this anywhere else in the Harmony code. 

>Or is that logic with the file naming in the python service library?

Yup.